src/dbus_api/api.rs

```diff
@@ -65,9 +65,9 @@ fn get_next_arg<'a, T>(iter: &mut Iter<'a>, loc: u16) -> Result<T, MethodErr>
 }
 
 /// Get filesystem name from object path
-fn object_path_to_pair(dbus_context: &DbusContext,
-                       fs_object_path: &dbus::Path)
-                       -> Result<(Uuid, Uuid), (MessageItem, MessageItem)> {
+fn fs_object_path_to_pair(dbus_context: &DbusContext,
+                          fs_object_path: &dbus::Path)
+                          -> Result<(dbus::Path<'static>, Uuid), (MessageItem, MessageItem)> {
```

Comment:
comments need updating.

style: Hmm, do you think it's worthwhile to use `type`s to help clarify what exactly the `Path<'static>` *is*? We could make it more clear *which* uuid we're returning in a similar manner. Of course the comments can describe but `type`s may be less likely to get stale. Thoughts?